core/deployment/src/main/java/io/quarkus/deployment/steps/NativeImageAutoFeatureStep.java

```diff
@@ -62,44 +53,25 @@
     private static final MethodDescriptor RERUN_INITIALIZATION = ofMethod(
             "org.graalvm.nativeimage.impl.RuntimeClassInitializationSupport",
             "rerunInitialization", void.class, Class.class, String.class);
+    private static final MethodDescriptor RESOURCES_REGISTRY_ADD_RESOURCES = ofMethod(
+            "com.oracle.svm.core.configure.ResourcesRegistry",
+            "addResources", void.class, String.class);
+    private static final MethodDescriptor RESOURCES_REGISTRY_IGNORE_RESOURCES = ofMethod(
+            "com.oracle.svm.core.configure.ResourcesRegistry",
+            "ignoreResources", void.class, String.class);
     static final String RUNTIME_REFLECTION = RuntimeReflection.class.getName();
     static final String JNI_RUNTIME_ACCESS = "com.oracle.svm.core.jni.JNIRuntimeAccess";
     static final String BEFORE_ANALYSIS_ACCESS = Feature.BeforeAnalysisAccess.class.getName();
     static final String DYNAMIC_PROXY_REGISTRY = "com.oracle.svm.core.jdk.proxy.DynamicProxyRegistry";
     static final String LOCALIZATION_FEATURE = "com.oracle.svm.core.jdk.LocalizationFeature";
 
-    @BuildStep
-    List<NativeImageResourceBuildItem> registerPackageResources(
```

Comment:
Why do you remove this method? `NativeImageResourceDirectoryBuildItem` does not seem to be related to the `-H:IncludeResources` problem, or?

Reply:
Ah, I see the method was just moved..

Reply:
Yes, I only move it to another class, where I add another related method.